tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -826,6 +828,72 @@ class WskBasicUsageTests
                 s"(?i)action /${namespace}/${actNameNoDescOrParams}\\s*\\(parameters: none defined\\)")
     }
 
+    it should "save action code to file" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "saveAction"
+            val seqName = "seqName"
+            val dockerName = "dockerName"
+            val containerName = s"bogus${Random.alphanumeric.take(16).mkString.toLowerCase}"
+            val saveName = s"save-as-$name.js"
+            val currentDir = System.getProperty("user.dir")
+            val saveDir = if (currentDir.endsWith("tests")) {
+                currentDir
+            } else {
+                s"$currentDir${File.separator}tests"
+            }
+            val badSaveName = s"bad-directory${File.separator}$saveName"
+
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) =>
+                    action.create(name, defaultAction)
+            }
+
+            wsk.action.get(name, save = Some(true)).stdout should include(s"saved action code to $name.js")
```

Comment:
i see the difference --- i think for simplicity, i'd prefer one option.
is this cobra getting in the way again because you can't have a default argument?

ie `--save` saves to inferred name and `--save file.name` to save to explicit file.

Reply:
Unfortunately Cobra will not allow two different data types for the same flag.